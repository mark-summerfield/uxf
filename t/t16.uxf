uxf 1.0 One Column Table
[= <Countries> <Name> =
<Netherlands>
<Wales>
<Belgium>
<Australia>
<Japan>
<Ukraine>
=]
