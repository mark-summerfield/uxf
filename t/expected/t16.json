{
  "UXF^table": {
    "comment": null,
    "name": "Countries",
    "fields": {
      "Name": null
    },
    "records": [
      [
        "Netherlands"
      ],
      [
        "Wales"
      ],
      [
        "Belgium"
      ],
      [
        "Australia"
      ],
      [
        "Japan"
      ],
      [
        "Ukraine"
      ]
    ]
  }
}